### PR TITLE
Create initial schema sql scripts for hwsc-user-svc

### PR DESCRIPTION
**Pull request by kimlisa**

### CODE REVIEW

File: int/hwsc-user-svc/test_create.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+docker rm -f devtest;\
+docker run --name devtest --mount type=bind,source="$(pwd)",target=/sql -d postgres:alpine;\
+sleep 5;\
+docker exec -it devtest psql -U postgres -c "CREATE DATABASE testdb;";\
+docker exec -it devtest psql -U postgres -f /sql/S1_CREATE_DOMAINS.sql;\
+docker exec -it devtest psql -U postgres -f /sql/S2_CREATE_USER_TABLE.sql;\
+docker exec -it devtest psql -U postgres -f /sql/S3_CREATE_DOCUMENT_TABLE.sql;\
+docker exec -it devtest psql -U postgres -f /sql/S4_CREATE_SHARED_TABLE.sql;\
```
Comment (faraonc) [inline int/hwsc-user-svc/test_create.sh]: I do not think this script is still needed. The script names are also wrong.

File: int/hwsc-user-svc/test_insert.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+INSERT INTO user_account (uuid, first_name, last_name, email, password, organization, create_timestamp)
```
Comment (faraonc) [inline int/hwsc-user-svc/test_insert.sql]: The schema is different now.

**Review by faraonc — CHANGES_REQUESTED** [review]

File: int/hwsc-user-svc/test_create.sh (hunk 1 above)
Comment (kimlisa) [reply int/hwsc-user-svc/test_create.sh]: Updated the script, this script is just for fast validation that table creation and inserts work as expected and view the table afterwards. Should I move it to gitignore?

File: int/hwsc-user-svc/test_insert.sql (hunk 2 above)
Comment (kimlisa) [reply int/hwsc-user-svc/test_insert.sql]: I set defaults so I don't have to set it in user-service, unless i have to for some reason:
is_verified should always be false on insert
created_date is defaulted to current date on insert time

should i be explicitly setting it in user-svc instead?

File: int/hwsc-user-svc/test_create.sh (hunk 1 above)
Comment (faraonc) [reply int/hwsc-user-svc/test_create.sh]: we can ignore this file.

File: int/hwsc-user-svc/test_insert.sql (hunk 2 above)
Comment (faraonc) [reply int/hwsc-user-svc/test_insert.sql]: We should always manually set values in the service. Relying on a db makes it hard to debug later on.

**Review by faraonc — APPROVED** [review]

**Comment by faraonc** [comment]
TU, you can merge without another reviewer.

**Pull request merged**